pkg/serverless/serverless.go

```diff
@@ -211,6 +211,7 @@ func handleInvocation(doneChannel chan bool, daemon *daemon.Daemon, arn string,
 	log.Debug("Received invocation event...")
 	daemon.ExecutionContext.SetFromInvocation(arn, requestID)
 	daemon.ComputeGlobalTags(config.GetConfiguredTags(true))
+	daemon.StartLogCollection()
```

Comment:
This addition is confusing to me. Don't we expect that log collection has already started by this point? Also, this `handleInvocation` is called with every invocation, which worries me that we are trying to start logs collection multiple times?